From af58d027aaab13a1d9c234964e27a176c4a58d42 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 1 Nov 2022 19:00:25 +0800
Subject: [PATCH] dts: Aptiv: disable dma coherent for pfe for cut 1.1
 compat

This patch comes from:

meta-aptiv/recipes-aptiv/linux-s32/files/0001-disable-dma-coherent-for-pfe-for-cut-1.1-compat.patch

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32g-pfe.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi b/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
index d7acf4316948..d69f8ba36d15 100644
--- a/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
@@ -38,7 +38,7 @@ pfe: pfe@46000000 {
 				 <&serdes1 PHY_TYPE_XPCS 1 1>,
 				 <&serdes0 PHY_TYPE_XPCS 1 1>;
 			phy-names = "emac0_xpcs", "emac1_xpcs", "emac2_xpcs";
-			dma-coherent;
+			/* dma-coherent; Disable for cut 1.1 compatibility OBE */
 			memory-region = <&pfe_reserved_bmu2>, <&pfe_reserved_rt>,
 					<&pfe_reserved>, <&pfe_reserved_bdr>;
 			memory-region-names = "pfe-bmu2-pool", "pfe-rt-pool",
-- 
2.36.1

